templates/engage/index.html

```diff
@@ -1446,6 +1446,19 @@ <h4>
         <p class="u-no-padding--bottom u-hide--small">WSLConf is a community-organized event on all things Windows Subsystem for Linux and WSL-related. The event will include two days of workshops, presentations, and networking events for developers, sysadmins, and power users on WSL.</p>
       </div>
     </div> <!-- 1st -->  
+    <div class='col-4 blog-p-card--post'>
```

Comment:
These should be an `article` otherwise the headers will confuse search engines. Feel free to file issue as it's out of scope for this branch

Reply:
Nice - filed an issue [here](https://github.com/canonical-web-and-design/ubuntu.com/issues/6295), will sort that shortly.